automation_tools/__init__.py

```diff
@@ -1861,31 +1854,21 @@ def upstream_install(admin_password=None, run_katello_installer=True):
         return installer_options
 
 
-def downstream_install(admin_password=None, run_katello_installer=True):
-    """Task to install Satellite 6
-
-    The following environment variables affect this command:
-
-    ADMIN_PASSWORD
-        Optional, defaults to 'changeme'. Foreman admin password.
-    BASE_URL
-        URL for the compose repository.
-    SATELLITE_VERSION
-        Satellite version.
+def downstream_install(admin_password='changeme', run_katello_installer=True,
+                       sat_version='6.6', url=None):
+    """Task to install Satellite from internal repo
 
+    :param str sat_version: Indicates which Satellite version is being installed
+    :param str url: the repo URL where to install from
```

Comment:
New options are missing in the docstring.

Reply:
Old options are missing, gonna fill 'em up